### DISCUSSION TITLE
Why are there multiple places to define workspace config?

### DISCUSSION THREAD

**Discussion by lamualfa** [vercel/turborepo#1878] [Help] [answered]
Where's the correct place should be to use?

**First place is on `package.json`**

![image](https://user-images.githubusercontent.com/39755201/188815027-f80d51e9-989b-4c4b-9eb8-b555253fa75a.png)

**Second place is on `pnpm-workspace.yml`**

![image](https://user-images.githubusercontent.com/39755201/188815158-8121471a-8d7c-44ae-b5e3-0151a973a37c.png)

**Comment by mehulkar** [comment, marked as answer]
This is not something defined by turborepo. The first screenshot you posted is how `npm` defines workspaces, the second is how `pnpm` defines workspaces. You only need one of them, depending on whether you are using npm or pnpm.

**Reply by lamualfa** [reply]
Can I remove the workspace config inside package.json and use the `pnpm-workspace.yml` only when using PNPM?

**Reply by nathanhammond** [reply]
Yes.
